src/buffer.jl

```diff
@@ -202,7 +202,9 @@ function copydata!(data::Ptr{UInt8}, buf::Buffer, nbytes::Integer)
 end
 
 # Insert data to the current buffer.
-function insertdata!(buf::Buffer, data::Ptr{UInt8}, nbytes::Integer)
+# `data` must not alias `buf`
+function insertdata!(buf::Buffer, data::Union{AbstractArray{UInt8}, Memory})
```

Comment:
This change looks breaking. Is this public API?

Reply:
No, `insertdata!` is internal and not part of the docs.

https://juliahub.com/ui/Search?q=insertdata!&type=code